Tensile/Source/client/source/PerformanceReporter.cpp

```diff
@@ -148,13 +146,13 @@ namespace Tensile
             m_memBandwidthMBps = std::numeric_limits<double>::quiet_NaN();
         }
         
-        void PerformanceReporter::setPerfModel(double l2ReadHits, double l2WriteHits, double l2ReadBwMultiplier, double readEff, int opsPerCycle)
+        void PerformanceReporter::setPerfModel(double l2ReadHits, double l2WriteHits, double l2ReadBwMultiplier, double readEff)
         {
             m_l2ReadHits = l2ReadHits;
             m_l2WriteHits = l2WriteHits;
             m_l2ReadBwMul = l2ReadBwMultiplier;
             m_readEff = readEff;
-            m_ops = opsPerCycle;
+            m_ops = 64;
```

Comment:
is m_ops now dead?  Seems we should remove.

Reply:
it's not dead- we need it to calculate aluUs. I can just pass in 64 if the name m_ops is a problem.